Modules/faulthandler.c

```diff
@@ -350,9 +353,18 @@ faulthandler_fatal_error(int signum)
     /* restore the previous handler */
     faulthandler_disable_fatal_handler(handler);
 
-    PUTS(fd, "Fatal Python error: ");
-    PUTS(fd, handler->name);
-    PUTS(fd, "\n\n");
+    if (found) {
+        PUTS(fd, "Fatal Python error: ");
+        PUTS(fd, handler->name);
+        PUTS(fd, "\n\n");
+    }
+    else {
+        char unknown_signum[255] = {0,};
```

Comment:
255 bytes is overkill, use 23. a signed 64-bit int can't be more than 20-21 chars.

Reply:
Thanks nice calculation!